src/components/implementation/no_interface/vkernel/vk_types.h

```diff
@@ -11,14 +14,14 @@
 #define VM_SHM_SZ       (1<<20)	        /* Shared memory mapping for each vm = 4MB */
 #define VM_SHM_ALL_SZ   ((VM_COUNT>0)?(VM_COUNT*VM_SHM_SZ):VM_SHM_SZ)
 
-#define VM_BUDGET_FIXED 400000
-#define VM_PRIO_FIXED   TCAP_PRIO_MAX
+#define VM_FIXED_PERIOD_MS 10
+#define VM_FIXED_BUDGET_MS 5
 
 enum vm_captbl_layout {
-	VM_CAPTBL_SELF_EXITTHD_BASE    = BOOT_CAPTBL_FREE,
+	VM_CAPTBL_SELF_SINV_BASE       = BOOT_CAPTBL_FREE,
 
 	/* VM1~ I/O Capabilities layout */
-        VM_CAPTBL_SELF_IOTHD_BASE      = round_up_to_pow2(VM_CAPTBL_SELF_EXITTHD_BASE + NUM_CPU_COS*CAP16B_IDSZ, CAPMAX_ENTRY_SZ),
+        VM_CAPTBL_SELF_IOTHD_BASE      = round_up_to_pow2(VM_CAPTBL_SELF_SINV_BASE + NUM_CPU_COS*CAP32B_IDSZ, CAPMAX_ENTRY_SZ),
```

Comment:
I've used mainly how `BOOT_CAPTBL_SELF_XXX_BASE` in `cos_types.h` were setup as reference, this has been there from the first implementation of `vkernel`. 

I totally agree that it's not portable and I'll change. Thank you!